Blish HUD/GameServices/Input/Keyboard/KeyboardHandler.cs

```diff
@@ -125,23 +130,19 @@ public class KeyboardHandler : IInputHandler {
         private void EndTextInputAsyncInvoke(IAsyncResult asyncResult) { _textInputDelegate?.EndInvoke(asyncResult); }
 
         private bool ShouldBlockKeyEvent(Keys key) {
-            // Prevent blocking shift for input capitalization
-            // if (key == Keys.LeftShift || key == Keys.RightShift) return false; // "SHIFT" support temporarily disabled
+            // TODO: WIN key combinations should probably completely handled by the OS
 
             // Skip keys that we wish to explicitly ignore
             if (_hookIgnoredKeys.Contains(key)) return false;
 
-            // Prevent blocking alt + x modifier
-            if (_keysDown.Contains(Keys.LeftAlt) || _keysDown.Contains(Keys.RightAlt)) return false;
-
             return true;
         }
 
         private bool ProcessInput(KeyboardEventType eventType, Keys key) {
             _inputBuffer.Enqueue(new KeyboardEventArgs(eventType, key));
 
             if (_textInputDelegate != null) {
-                string chars = TypedInputUtil.VkCodeToString((uint)key, eventType == KeyboardEventType.KeyDown);
+                string chars = TypedInputUtil.vkCodeToString((uint)key, eventType == KeyboardEventType.KeyDown);
```

Comment:
Since this function is accessible outside of the class.  Let's keep the name as `VkCodeToString` (capital `V`).

Reply:
Good catch.